tests/lib/muinstaller/main.go

```diff
@@ -295,6 +297,21 @@ func postSystemsInstallFinish(cli *client.Client,
 	return waitChange(chgId)
 }
 
+func maybeGetOptionalInstall() *client.OptionalInstallRequest {
+	f, err := os.Open("./optional-install.json")
```

Comment:
I think it would be better to pass the json file as a program argument to make it more explicit.

Reply:
but agreed on this